<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-F3I22UON" ignored="1" name="Regression" tests="6" failures="1" timestamp="2020-08-20T04:00:49 PDT" time="0.119" errors="0">
  <testcase name="Demo1" time="0.007" classname="testNG.ClassTNG1"/>
  <testcase name="Demo1" time="0.003" classname="testNG.ClassTNG1"/>
  <testcase name="Demo1" time="0.003" classname="testNG.ClassTNG1"/>
  <testcase name="WebHomeLogin" time="0.006" classname="testNG.ClassTNG2"/>
  <testcase name="Login" time="0.004" classname="testNG.ClassTNG2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.ClassTNG2.Login(ClassTNG2.java:19)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase name="AppHomeLogin" time="0.006" classname="testNG.ClassTNG2"/>
  <testcase name="APIHomeLogin" time="0.0" classname="testNG.ClassTNG2">
    <ignored/>
  </testcase> <!-- APIHomeLogin -->
</testsuite> <!-- Regression -->
